### ISSUE TITLE
Exposing Batch norm flag

### ISSUE THREAD

**Issue by jdeschamps** [CAREamics/careamics#471]
## Problem

There may be issues with BatchNorm layers when there are many background patches. Let's investigate, but we may benefit from allowing BatchNorm to be disabled in the UNet.
